src/python/dxpy/scripts/dx.py

```diff
@@ -2512,23 +2512,12 @@ def install(args):
         err_exit()
 
 def uninstall(args):
-    app_desc = get_app_from_path(args.app)
```

Comment:
I think this original flow is still necessary because we want to support users specifying the app by path. Please bring this back in. I apologize if my previous comment misled you.